bucket/archwsl.json

```diff
@@ -36,6 +36,7 @@
     ],
     "uninstaller": {
         "script": [
+            "if ($cmd -eq 'uninstall') { return }",
```

Comment:
```suggestion
            "if ($cmd -eq 'update') { return }",
```

Reply:
You’r correct. Thanks for catching that.